### PR TITLE
feat: implement missing exports [cloudbuild]

### PR DESCRIPTION
**Pull request by acpana**
Trying to implement any missing Exports in the direct package.

### CODE REVIEW

**Comment by justinsb** [comment]
Looks good.  I know you say "implement any missing exports" but it's much easier to review these one at a time I think.  I'm not sure why the tests aren't failing here though, I would expect to see that we are now generating an exported version?  Maybe we need to add support to the export function in unified_test.go ?

**Comment by acpana** [comment]
Going to take up Justin on having separate PRs for these. This one is for `cloudbuild`. The dynamic (postsubmit) tests should pass now 💯

**Review by yuwenma — COMMENTED** [review]
Thanks for adding the export function. Could you generate the golden export file so we can validate?

File: pkg/controller/direct/cloudbuild/workerpool_controller.go [hunk 1]
```diff
@@ -142,6 +142,29 @@ func (m *model) AdapterForObject(ctx context.Context, reader client.Reader, u *u
 }
 
 func (m *model) AdapterForURL(ctx context.Context, url string) (directbase.Adapter, error) {
+	// Format: //cloudbuild.googleapis.com/projects/<project>/lcoations/<location>/workerPools/<id>
```
Comment (yuwenma) [inline pkg/controller/direct/cloudbuild/workerpool_controller.go:145]: can we build this from the existing `CloudBuildWorkerPoolIdentity`?

Comment (acpana) [reply pkg/controller/direct/cloudbuild/workerpool_controller.go:145]: Chatted offline, we're going to punt on this for a bit as return nil, nil is really important to the registry.Export func

File: pkg/test/resourcefixture/testdata/basic/cloudbuild/v1beta1/cloudbuildworkerpool/_generated_export_cloudbuildworkerpool.golden [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    networkConfig:
+      egressOption: NO_PUBLIC_EGRESS
+      peeredNetworkIPRange: /29
+      peeredNetworkRef:
+        external: projects/${projectId}/global/networks/computenetwork-${uniqueId}
+    workerConfig:
+      diskSizeGb: 100
+      machineType: e2-medium
+  projectRef:
+    name: ${projectId}
```
[the remark this answers is not in the document]
Comment (acpana) [reply pkg/test/resourcefixture/testdata/basic/cloudbuild/v1beta1/cloudbuildworkerpool/_generated_export_cloudbuildworkerpool.golden:18]: That makes sense! Building this into the generated code then -- https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/2941

**Review by yuwenma — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**